temporian/core/data/sampling.py

```diff
@@ -37,3 +39,6 @@ def creator(self):
 
     def set_creator(self, creator):
         self._creator = creator
+
+    def is_unix_timestamp(self) -> bool:
```

Comment:
Can this be a @property as it doesn't receive any attributes?

Reply:
Not addressing as part of this PR since all of `Sampling`'s methods already not properties, and https://github.com/google/temporian/issues/46 is open to deal with this globally